arabic/dream/dream.md

```diff
@@ -2,4 +2,5 @@
 احلم......الاحلام تتحقق اذا صدقة النية
 ولكن الاحلام لا تتحقق الا بالتوكل على الله والاخذ بالاسباب
 الان انت فى بداية الطريق ماذا ستفعل
-1- [ستبدأ فى التعلم] (https://github.com/udacity/create-your-own-adventure/blob/master/arabic/learn/learn.md
+1- [ستبدأ فى التعلم] (https://github.com/udacity/create-your-own-adventure/blob/master/arabic/learn/learn.md)
+2-[ستبدأ بالعمل](https://raw.githubusercontent.com/eyadsalama/create-your-own-adventure/93dfdcffd1067a7a25a2baa513aa2b53f2436fe3/arabic/work.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.